sim/core/buffs.go

```diff
@@ -2211,16 +2220,6 @@ func CreateExtraAttackAuraCommon(character *Character, buffActionID ActionID, au
 		},
 	})
 
-	MakePermanent(character.GetOrRegisterAura(Aura{
-		Label:     "Extra Attacks  (Main Hand)", // Tracks Stored Extra Attacks from all sources
-		ActionID:  ActionID{SpellID: 21919},     // Thrash ID
-		Duration:  NeverExpires,
-		MaxStacks: 4, // Max is 4 extra attacks stored - more can proc after
-		OnInit: func(aura *Aura, sim *Simulation) {
-			aura.Unit.AutoAttacks.mh.extraAttacksAura = aura
-		},
-	}))
```

Comment:
Deleting this may cause problems and might be why you're seeing differences in extra attacks. Should we just remove `MakePermanent` here and set the Duration + OnReset instead?
```
character.GetOrRegisterAura(Aura{
		Label:     "Extra Attacks  (Main Hand)", // Tracks Stored Extra Attacks from all sources
		ActionID:  ActionID{SpellID: 21919},     // Thrash ID
		Duration:  NeverExpires,
		MaxStacks: 4, // Max is 4 extra attacks stored - more can proc after
    OnReset: func(aura *Aura, sim *Simulation) {
      aura.activate(sim)
    },
		OnInit: func(aura *Aura, sim *Simulation) {
			aura.Unit.AutoAttacks.mh.extraAttacksAura = aura
		},
	}))
```

Reply:
Nah, I saw the diff long before I removed this (and the diff was only 0.5 extra attacks on 100k iterations).
I added logging in the OnInit here and it never got ran so all this whole block does is add another `MakePermanent` on the Thrash-auras here https://github.com/wowsims/sod/blob/b3b7861ef61ecd8108eb92feca15dc610da9df6a/sim/core/attack.go#L565